go/stellar/stellar.go

```diff
@@ -951,16 +968,11 @@ func claimPaymentWithDetail(ctx context.Context, g *libkb.GlobalContext, walletS
 		// Direction from caller
 		useDir = *dir
 	}
-	sp := NewSeqnoProvider(libkb.NewMetaContext(ctx, g), walletState)
-	// Throw a random ID into the transaction memo so that we get a new txID each time.
-	// This makes it easy to resubmit without hitting a txID collision on the server.
-	memoID, err := randMemoID()
-	if err != nil {
-		return res, err
-	}
-	g.Log.CDebugf(ctx, "using randomized memo ID: %v", memoID)
+	mctx := libkb.NewMetaContext(ctx, g)
+	sp := NewSeqnoProvider(mctx, walletState)
+	tb := getTimeboundsForSending(mctx)
```

Comment:
looks like we can achieve the same thing using timebounds, that are signed into the transaction and change txid. Unless we plan to retry these more than one time in given second

Reply:
Yes I like this plan of getting rid of the randomized memo because timebounds also cause a different txID each time. I don't think the 1 second resolution should be a problem in this case.